hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/TracingContext.java

```diff
@@ -153,6 +153,24 @@ public static String validateClientCorrelationID(String clientCorrelationID) {
     return clientCorrelationID;
   }
 
+  public void setPrimaryRequestIDBlob() {
+    // Generate a UUID, remove dashes, and shuffle by selecting a random 8-character substring
+    String uuid = UUID.randomUUID().toString().replace("-", "");
+    StringBuilder randomId = new StringBuilder(8);
+
+    for (int i = 0; i < 8; i++) {
```

Comment:
Can we create a separate method for this and place it in some utility class? Generating a constant-length substring from a given string should be treated as a separate method.

Reply:
taken, and since this class is already part of the utils package, have added method to the same class